brands/amenity/restaurant.json

```diff
@@ -1744,8 +1747,9 @@
     "tags": {
       "amenity": "restaurant",
       "brand": "ガスト",
-      "brand:en": "Gusto",
+      "brand:en": "びっくりドンキー",
```

Comment:
This tag is brand:LANG. This was brand:en before since it was in English. I think you want to update the lang for this tag value now.

Reply:
Thanks for catching that. This was an unintentional edit. It should be `"brand:en": "Gusto",`. I have now pushed a the fix.